Mlem/App/Views/Pages/MessageFeedView/MessageBubbleView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  MessageBubbleView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 2024-12-22.
+//
+
+import LemmyMarkdownUI
+import MlemMiddleware
+import SwiftUI
+
+struct MessageBubbleView: View {
+    @Environment(NavigationLayer.self) var navigation
+    @Environment(Palette.self) var palette
+    
+    let message: any Message
+    
+    var body: some View {
+        Group {
+            let blocks: [BlockNode] = .init(message.content)
+            if blocks.isSimpleParagraphs {
+                MarkdownText(blocks, configuration: message.isOwnMessage ? .inverted : .default)
+            } else {
+                Markdown(blocks, configuration: message.isOwnMessage ? .inverted : .default)
+            }
+        }
+        .padding(Constants.main.standardSpacing)
+        .padding(message.isOwnMessage ? .trailing : .leading, 7)
```

Comment:
*nit*

This `message.isOwnMessage ? .trailing : .leading` phrase is repeated a _lot_--probably nicer to extend `Message` to provide this as something like `ownerAwareAlignment`

Reply:
Yeah, I did try this but unfortunately half of the types are `Alignment` and the other half are `HorizontalAlignment`, so we'd need two computed properties. And in some cases I want `.leading` for `isOwnMessage`, and for others I need `.trailing`... so we'd either need 4 computed properties or some sort of `flip()` method of `Alignment` and `HorizontalAlignment` (which don't exist yet so I'd need to implement them). I thought it would be simpler to do it this way.

Reply:
Makes sense to me 👍